interface/orders/single_order_results.inc.php

```diff
@@ -63,23 +65,38 @@ function storeNote($s)
     return $key;
 }
 
+function diagDesc($code)
+{
+    if (!$code) {
+        return false;
+    }
+    $tmp = explode(':', $code);
+    $code = $tmp[1];
+    $query = "SELECT CONCAT('ICD10:',formatted_dx_code) AS code, short_desc FROM icd10_dx_order_code ";
+    $query .= "WHERE active = 1 AND valid_for_coding = 1 AND (formatted_dx_code LIKE '" . $code . "%') ";
+    $result = sqlQuery($query);
```

Comment:
We have standardized code type functions. See this function (lookup_code_descriptions) to collect descriptions and examples of use in codebase:
https://github.com/openemr/openemr/blob/master/custom/code_types.inc.php#L680

Reply:
only for you Brady...